include/dbclean.php

```diff
@@ -18,13 +18,11 @@ function dbclean_run(&$argv, &$argc) {
 	}
 
 	if ($stage == 0) {
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 1);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 2);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 3);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 4);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 5);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 6);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 7);
+		for ($i = 1; $i <= 7; $i++) {
+			if (!Config::get('system', 'finished-dbclean-'.$i)) {
```

Comment:
It seems like you never reset the `finished-dbclean-X` values to `false` after they've been set to `true`, so the `dbclean` task can only be ran once, all subsequent runs will skip all but the first step. Is that intended?